src/components/modules/table-content/components/table-content-trigger.tsx

```diff
@@ -0,0 +1,24 @@
+'use client';
+
+import type { TableContentProps } from '../table-content.types';
+
+import { Link } from '@/components/elements/link';
+import { Text } from '@/components/elements/text';
+
+const TextComponent = ({ children }: Component) => (
+  <Text fontSize="m" fontWeight="bold" color="white">
+    {children}
+  </Text>
+);
+
+export const TableContentTriggerComponent = ({ text, href }: TableContentProps['links'][0]) => {
```

Comment:
https://www.typescriptlang.org/docs/handbook/2/indexed-access-types.html
```suggestion
export const TableContentTriggerComponent = ({ text, href }: TableContentProps['links'][number]) => {
```